client/src/components/EditProfile/EditPhoto/EditPhoto.tsx

```diff
@@ -1,77 +1,90 @@
-  }
+  const handleCapture = ({ target }: React.ChangeEvent<HTMLInputElement>) => {
+    const files = target.files;
+    if (files) {
+      uploadImage({ file: files[0] })
+        .then(() => {
+          fetchProfileAndUpdateContext();
+          updateSnackBarMessage('Your photo is uploaded successfully');
+        })
+        .catch(() => ({
+          error: { message: 'Unable to connect to server. Please try again' },
+        }));
+    }
+  };
 
-  function removeFile() {
-    setFile('');
+  async function removeFile() {
+    const profile = loggedInUserProfile;
+    if (profile) {
+      profile.photo = '';
+      await editProfile(profile);
+      fetchProfileAndUpdateContext();
+    }
   }
 
   return (
     <Grid container component="main" className={classes.root}>
-      <Grid item xs={12} sm={8} md={7} elevation={4} component={Paper} square style={{ height: 560 }}>
-        <Box
-          display="flex"
-          alignItems="flex-start"
-          justifyContent="space-between"
-          flexDirection="column"
-          minHeight="100%"
-          paddingBottom={40}
-        >
-          <Box width="100%" maxWidth={450} p={3} alignSelf="center">
-            <Grid container>
-              <Grid item xs>
-                <Typography className={classes.title} component="h1" variant="h5">
-                  Profile Photo
-                </Typography>
-              </Grid>
+      <Box
+        display="flex"
+        alignItems="flex-start"
+        justifyContent="space-between"
+        flexDirection="column"
+        minHeight="100%"
+        paddingBottom={40}
+      >
+        <Box width="100%" maxWidth={450} p={3} alignSelf="center">
+          <Grid container>
+            <Grid item xs>
+              <Typography className={classes.title} component="h1" variant="h5">
+                Profile Photo
+              </Typography>
             </Grid>
-            <Box display="flex" alignItems="center" justifyContent="center">
-              <Avatar src={file} className={classes.avatar} />
-            </Box>
-            <Box display="flex" alignItems="center" justifyContent="center">
-              <Typography className={classes.reminder} component="h4" variant="h6">
-                Be sure to use a photo that clearly shows your face
+          </Grid>
+          <Box display="flex" alignItems="center" justifyContent="center">
+            <Avatar src={loggedInUserProfile?.photo} className={classes.avatar} />
+          </Box>
+          <Box display="flex" alignItems="center" justifyContent="center">
+            <Typography className={classes.reminder} component="h4" variant="h6">
+              Be sure to use a photo that clearly shows your face
+            </Typography>
+          </Box>
+          <Box className={classes.centerRow}>
+            <Button component="label" className={classes.uploadButton} variant="outlined" color="primary">
+              Upload a file from your device
+              <input
```

Comment:
Use MUI `TextField` component for this `input`

Reply:
I have tried. However, there are some features that TextField doesn't support.